### DISCUSSION TITLE
Is it possible to use this solution to read a ~150mb 682k word 4million character pdf file ?

### DISCUSSION THREAD

**Discussion by gt732** [mayooear/ai-pdf-chatbot-langchain#89] [General]
Trying to see if it is possible to create a chatbot with a pdf file that is 3000 pages long and ~150mb in size.

**Comment by dalkommatt** [comment]
Have you tried it? https://docs.pinecone.io/docs/indexes s1 pods have capacity "for around 5M vectors of 768 dimensions" so I'd say it should definitely be possible, not sure how long it would take to answer though. If you have problems ingesting you could try breaking the PDF into smaller chunks too.
